test-utils/test-utils-core/src/test/java/uk/gov/justice/services/test/utils/core/random/RandomGeneratorTest.java

```diff
@@ -178,6 +180,26 @@ public void shouldGenerateRandomBackwardDate() {
     }
 
     @Test
+    public void shouldGenerateRandomForwardZonedDateTime() {
+        final ZonedDateTime startDateTime = ZonedDateTime.now(UTC);
```

Comment:
Not sure if we want to use our framework Clock to create zoneDateTimes to try and be consistent.